modules/programs/pylint.nix

```
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.pylint;
+  listToValue = concatMapStringsSep ",\n    " (generators.mkValueStringDefault { });
+  iniFormat = pkgs.formats.ini { inherit listToValue; };
+in {
+  meta.maintainers = [ hm.maintainers.florpe ];
+  options.programs.pylint = {
+    enable = mkEnableOption "The pylint Python linter.";
```

```suggestion
    enable = mkEnableOption "the pylint Python linter";
```

`mkEnableOption` automatically appends a final period.